File: squad-bsc_raneem_guessing game.c
#include <stdio.h>
int main() {
  int secretnumber=5;
  int guess;
  int guesscount=0;
  int guesslimit=4;
  int outofrange=0;
  
  while (secretnumber!= guess && outofrange ==0)
  { if(guesscount<guesslimit){
  printf("enter a number : ");
  scanf("%d",&guess);
  guesscount++;
  }else
  outofrange=1;}
  
  if (outofrange==1){
      printf("out of range");
  }else
  printf("you win");
   return 0;
}

File: squad-bsc_raneem_guessing_game.py
secret_word = "raneem"
guess = ""
guess_count = 0
guess_limit = 3
out_of_guesses = False

while guess != secret_word and not (out_of_guesses):
    if guess_count < guess_limit:
        guess = input("please enter your guessing word: ")
        print('you have ' + str(guess_limit - 1 - guess_count) + ' turns left')
        guess_count += 1
    else:
        out_of_guesses = True
if out_of_guesses:
    print("out of guesses, you lose :(")
else:
    print("you win :) ")


